image-tools-interface.md

```diff
@@ -0,0 +1,49 @@
+# Image Tools Interface
+
+The below document describes an interface for the OCI Image Tools project as desired by the OCI Certification Program working group (Cert WG.)
+
+Goals:
+* An engineer can verify a container image complies with the OCI Image Spec (https://github.com/opencontainers/image-spec/releases) 
+
+Inputs:
+* (REQUIRED): Path or reference to a container image
+* (OPTIONAL): If the user would like verbose output
+
+Outputs:
+* Number of cases run
+* Number of cases passed
+* Number of cases failed
+* List of cases failed
+* Version of OCI Image Tools
```

Comment:
Replying to comments from @vbatts and @wking:

>and maybe git commit of the tool?

I was thinking that 'version of the OCI Image Tools' would be sufficient since only released versions of the tools will be relevant for image validation.  

>and version of the OCI spec tested against.

I didn't think of this, but it makes sense.  I'll update the PR.

>And the timestamp the validation just ran.
I'm not sure this is needed since a released version of the OCI Image Tool, a released version of the OCI Image Spec, and the target image should have the same output every time.

>Also, presumably this would need to output the digests of the objects tested.

@vbatts Would you mind suggesting what this might look like?